src/js/modules/breakpointHelper.js

```diff
@@ -60,3 +62,49 @@ export const getCurrentScreenWidth = () => {
     // If no breakpoints have been set
     return false;
 };
+
+export const withinBreakpoint = breakpointString => {
```

Comment:
Think maybe `isWithinBreakpoint` would be a better name if it returns true/false.

Also, can you add a comment to the function, with what params it takes as input?